docs/rules/line-hitter.md

```diff
@@ -1,4 +1,13 @@
-@todo #242:30min Add documentation for the RuleAssertTrueHitter.
-The documentation should be added to the `docs/rules/assert-true-hitter.md` file.
-The documentation should contain the description of the rule and the
-examples of the correct and incorrect code.
+# Line hitter
+At first glance, the tests cover everything and code coverage tools confirm it with 100%, but in reality the tests merely hit the code, without doing any output analysis.
+
+## Example
+```java
```

Comment:
@l3r8yJ, it isn't clear from the description whether it is an example of how to **do** something or how **not to do** it. Could you provide two different examples, one labeled 'Bad' and the other 'Good' (or something similar)? This way, I will understand what to do more quickly without the need to read the text description.